src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/PermanentStorage.java

```diff
@@ -17,12 +18,14 @@
 
 	/**
 	 * This function delete all files from task in Permanent Storage.
-	 * 
+	 *
 	 * @param task task with files information to be deleted
 	 * @return boolean representation, success (true) or failure (false) to delete
 	 *         files
 	 * @throws Exception
 	 */
 	boolean delete(SapsImage task) throws PermanentStorageException;
 
-}
+	List<AccessLink> generateAccessLinks(String taskId) throws PermanentStorageException;
```

Comment:
Sync with this [version](https://github.com/ufcg-lsd/saps-engine/blob/feature/swift-generate-link/src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/PermanentStorage.java).

Reply:
Done